packages/bolt-storage-file/README.md

```diff
@@ -0,0 +1,25 @@
+# Bolt File Storage
+
+Bolt [conversation store](https://slack.dev/bolt/concepts#conversation-store) that uses your file system as a database. Useful for retaining conversation state through restarts of your App in development.
```

Comment:
let's write this as a sentence and maybe give a little more detail on the why/problem being solved? no strong feelings, here's a stab

```suggestion
Tool to maintain a Bolt [conversation store](https://slack.dev/bolt/concepts#conversation-store) in development using your file system as a database. Improves developer experience by persisting conversation state through App restarts.
```

Reply:
> Tool to maintain a Bolt

😛 Na it's litterally a conversation store.

https://github.com/IBM/slack-wrench/pull/2/files#diff-7dcefd9507c82601c7c829c82b1a3005R10

I'll take a stab at it though.